src/transaction.ts

```diff
@@ -61,7 +61,7 @@ export interface RequestOptions {
 }
 
 export interface CommitOptions {
-  requestOptions?: IRequestOptions;
+  requestOptions?: Omit<IRequestOptions, 'requestTag'>;
```

Comment:
I believe it should be 
```ts
requestOptions?: Pick<IRequestOptions, 'priority'>;
```
here

Reply:
No, I don't think that's correct. `CommitOptions` are used for blind writes such as `database.table('some-table').insert(..)`, and those should support setting a transaction tag.

Reply:
(After our conversation earlier today): I've removed `transaction-tag` from `CommitOptions`.